apps/docs/content/guides/platform/custom-domains.mdx

```diff
@@ -79,6 +79,8 @@ Required outstanding validation records:
 
 Add the records to your domains' DNS settings. Make sure to trim surrounding whitespace. Use a low TTL value so you can quickly change the records if you make a mistake.
 
+Some DNS registrars automatically append your domain name to the DNS entries being created. As such, creating a DNS record for `api.example.com` might instead create a record for `api.example.com.example.com`. In such cases, please remove the domain name from the records you're creating; as an example, you would create a TXT record for `api`, instead of `api.example.com`.
```

Comment:
```suggestion
Some DNS registrars automatically append your domain name to the DNS entries being created. For example, creating a DNS record for `api.example.com` might instead create a record for `api.example.com.example.com`. In such cases, remove the domain name from the records you're creating. In other words, create a TXT record for `api`, instead of `api.example.com`.
```